app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsPresenter.kt

```diff
@@ -44,7 +40,6 @@ class PaymentMethodsPresenter(
   private var cachedGamificationLevel = 0
   private var cachedFiatValue: FiatValue? = null
   private var cachedPaymentNavigationData: PaymentNavigationData? = null
-  private var closeToLevelUp: Boolean = false
   private var hasStartedAuth = false
 
   companion object {
```

Comment:
There is a typo in **private const val PAYMENTE_NAVIGATION_DATA = "payment_navigation_data"**

Can you fix it?